SECURITY.md

```diff
@@ -0,0 +1,21 @@
+# Security Policy
+
+This document explains the security policy of webpack-cli and how we intend to support webpack and webpack-cli.
+
+## Supported Versions
+
+webpack CLI is currently supporting webpack v4 and webpack v5. Security fixes are released in patches.
+
+| webpack version | webpack-cli version           | Supported          |
+| --------------- | ----------------------------- | ------------------ |
+| 4.x.0           | ^3.1.2                        | :white_check_mark: |
```

Comment:
We have to rewrite this bit. 
After webpack `4.20.0` => min version required is `3.1.1`


https://github.com/webpack/webpack/releases/tag/v4.20.0

Reply:
gotcha, added now